common/ieee754_float/f1.ksy

```diff
@@ -0,0 +1,22 @@
+meta:
+  id: f1
+  title: IEEE 754 Min-float
```

Comment:
* This particular entry is *not* IEEE754-compliant in any way, and illustrates *one* of possible one-byte long floating point numbers representation, [as per entry in Wikipedia](https://en.wikipedia.org/wiki/Minifloat).
* The same entry is named "minifloat", not "min-float"

Reply:
* Yes, it is not defined in IEEE 754. But it uses the same principles. How about `IEEE 754-like`?
* I don't remember why I have called it this way.